File: insertion/insertion.py
def insertionSort(arr):
	for i in range(len(arr)):
		preIndex = i - 1
		current = arr[i]
		while preIndex >= 0 and arr[preIndex] > current:
			arr[preIndex + 1] = arr[preIndex]
			preIndex -= 1
		arr[preIndex + 1] = current
	return arr

arr = [7,53,24,45,21,5,78,43,4,25,43]
rt = insertionSort(arr)

print(rt)

File: insertion/insertion.js
function insertionSort(arr) {
	var len = arr.length;
	var preIndex, current;
	for (var i = 1; i < len; i++) {
		preIndex = i - 1;
		current = arr[i];
		while (preIndex >= 0 && arr[preIndex] > current) {
			arr[preIndex + 1] = arr[preIndex];
			preIndex--;
		}
		arr[preIndex + 1] = current;
	}
	return arr;
}

var arr = new Array(7,53,24,45,21,5,78,43,4,25,43);

var rt = insertionSort(arr);

console.log(rt);